index.html

```diff
@@ -68,6 +68,174 @@ <h2 class="name">
+            </div>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="languages">
+                <li class="html">html</li>
+                <li class="css">css</li>
+                <li class="javascript">javascript</li>
+            </ul>
+            <button>See Project</button>
+        </section>
+
+    </section>
+
+    <section class="about-myself">
+        <h2 class="name myself">
+            About Myself
+        </h2>
+        <p class="about">
+            Hello I’m a software developer! I can help you build a product , feature
+             or website Look through some of my work 
+            and experience! If you like what you see and have a project you need coded, don’t hestiate
+             to contact me.
+        </p>
+ 
+            <a href="#" class="social-media">LET'S CONNECT</a>
+        
+        <ul class="social-media-links">
+            <li>
+                <a href="https://www.linkedin.com/feed/" class="twitter-icon icon-right">
+                    <img src="images/twitter.svg" alt="">
+                   </a> 
+            </li>
+            <li>
+                <a href="https://www.linkedin.com/feed/" class="angelList-icon icon-right">
+                    <img src="images/angelList.svg" alt="">
+                   </a>
+            </li>
+            <li>
+                <a href="https://www.linkedin.com/feed/" class="github-icon icon-right">
+                    <img src="images/github.svg" alt="">
+                   </a> 
+            </li>
+            <li>
+                <a href="https://www.linkedin.com/feed/" class="linkdin-icon icon-right">
+                    <img src="images/Linkedin-icon.svg" alt="">
+                   </a> 
+            </li>
+            <li>
+                <a href="https://www.linkedin.com/feed/" class="microsoft-icon icon-right">
+                    <img src="images/microsoft.svg" alt="">
+                   </a> 
+            </li> 
+           
+        </ul>
+        <button>Get My Resume</button>
+        
+        <ul class="lang">
+
+            <li>
+                <div class="arrow-down">
+                    <h4>Languages</h4>
+                    <a href="#">
+                        <img src="images/arrow-down-sign-to-navigate.png" alt="">
+                    </a>
+                </div>
+                <ul class="languages">
+                    <li class="javascript">
+                        <img src="images/Ellipse-1.svg" alt="">
+                        javascript
+                    </li>
+                    <li class="html">
+                        <img src="images/Ellipse-1-(1).svg" alt="">
+                        html
+                    </li>
+                    <li class="css">
+                        <img src="images/css.svg" alt="">
+                        css
+                    </li>
+                </ul>
```

Comment:
[OPTIONAL] I strongly recommend you type the languages' names properly (capitalization, etc). Having them lowercased makes your project look unprofessional.